python/graphstorm/eval/evaluator.py

```diff
@@ -341,6 +341,14 @@ def do_eval(self, total_iters, epoch_end=False):
             return True
         return False
 
+    def _get_early_stop_score(self, val_score):
```

Comment:
Since it looks like this is becoming part of the expected inner interface of the class (`do_early_stop` is public, and calls this inner function) can we add some type annotations so we know what to expect in the input arguments and return values? Then new implementers of this function will not what is the expected input/output.